src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/MetadataTypeTests.cs

```diff
@@ -252,17 +252,17 @@ public void MetadataTypeSymbolStruct01()
             Assert.Empty(compilation.GetDeclarationDiagnostics());
         }
 
-        [Fact]
+        [Fact(Skip = "PROTOTYPE")]
```

Comment:
I'm unsure what to do with this test. The test here revolves around the type `EventingTestBase`. That is a type that only exists in the beta version of the .NET Framework, it was never shipped in any RTM release. Hence there is no direct port of this to the standard reference assemblies. 

Is there an equivalent to this type that I can use in this test? Or if there isn't should we just delete the test? I'd much prefer finding an equivalent type to test against.

@AlekseyTs as you're most likely to have the context here.

Reply:
> Is there an equivalent to this type that I can use in this test? Or if there isn't should we just delete the test? I'd much prefer finding an equivalent type to test against.

From reading the original test, I believe it wasn't really intended to test something special about the ```EventProviderBase```. It looks like the goal was to verify the API behaviors around array types imported from metadata, and the ```EventProviderBase``` type conveniently referenced types like that. I think we can simply define the ```EventProviderBase```, as it used to be, in source in a separate compilation, then emit the compilation as image reference, add the reference, and test against that type instead. 

---
In reply to: [450993234](https://github.com/dotnet/roslyn/pull/45690#discussion_r450993234) [](ancestors = 450993234)

Reply:
Took that approach and it looks to have worked out well. Thanks.